test/test_linalg.py

```diff
@@ -7936,30 +7936,30 @@ def fn(torchfn, *args):
     def test_tensordot(self, device):
```

Comment:
Is this test good enough and doesn't need to be updated? Are all supported types of `dims` tested here?

Reply:
I think at least dtype parametrization should be added.

Reply:
good idea! Actually all the various possibilities were already tested by `test_jit.py` but it looks like a good idea to have a more direct test here. Fixed by 053beac580 and e7fc1ea4b1